From 022dbfe14478d8fb01d846c8ea4153560cfaf279 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Sat, 1 Jun 2013 12:55:03 -0500
Subject: [PATCH 5/7] Revert "regulator: twl: Remove another unused variable
 warning"

This reverts commit 029dd3cefa46ecdd879f9b4e2df3bdf4371cc22c.

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 drivers/regulator/twl-regulator.c |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/regulator/twl-regulator.c b/drivers/regulator/twl-regulator.c
index 74508cc..85ad355 100644
--- a/drivers/regulator/twl-regulator.c
+++ b/drivers/regulator/twl-regulator.c
@@ -650,6 +650,13 @@ static struct regulator_ops twl6030fixed_ops = {
 	.get_status	= twl6030reg_get_status,
 };
 
+static struct regulator_ops twl6030_fixed_resource = {
+	.enable		= twl6030reg_enable,
+	.disable	= twl6030reg_disable,
+	.is_enabled	= twl6030reg_is_enabled,
+	.get_status	= twl6030reg_get_status,
+};
+
 /*
  * SMPS status and control
  */
-- 
1.7.10.4

